<launch>
    <node name="rviz" pkg="rviz2" exec="rviz2" output="screen">
    </node>

    <node pkg="map_sensing" exec="map_sensing_node" name="map_sensing_node" output="screen">
    <param name="init_state_x" value="0.0"/>
    <param name="init_state_y" value="0.0"/>
    <param name="map/x_size"   value="50.0"/>
    <param name="map/y_size"   value="50.0"/>
    <param name="map/z_size"   value="5.0"/>
    <param name="map/obs_num"  value="30"/>
    <param name="map/resolution"  value="0.1"/>
    <param name="map/circle_num"  value="30"/>
    <param name="ObstacleShape/lower_rad" value="0.3"/>
    <param name="ObstacleShape/upper_rad" value="0.8"/>
    <param name="ObstacleShape/lower_hei" value="3.0"/>
    <param name="ObstacleShape/upper_hei" value="7.0"/>
    <param name="ObstacleShape/radius_l"  value="7.0"/>
    <param name="ObstacleShape/radius_h"  value="7.0"/>
    <param name="ObstacleShape/z_l"       value="7.0"/>
    <param name="ObstacleShape/z_h"       value="7.0"/>
    <param name="ObstacleShape/theta"     value="7.0"/>
    <param name="sensing/radius"          value="10.0"/>
    <param name="sensing/rate"            value="10.0"/>
    <param name="min_distance"            value="1.0"/>
    </node>
</launch>
